### ISSUE TITLE
Allow more than one model per language, consulted together

### ISSUE THREAD

**Issue by goldyfruit** [OpenVoiceOS/ovos-m2v-pipeline#257]
## Where it stands today

`_resolve_model_for_lang` resolves to exactly **one** model for a given language:

```
1. config["model"]        -- a single explicit override
2. config["models"]       -- {locale_or_lang: repo_id}, one repo per key
3. DEFAULT_MODELS         -- one repo per primary subtag
4. DEFAULT_MULTILINGUAL
```

So `config["models"]` already gives *different* models to *different* languages, but a single language can only ever name one. Everything downstream is built for that: `_SHARED_MODELS` is keyed by resolved path and already holds several distinct models in one process quite happily, and `load_shared_model` already shares one embedding between a classifier and a prototype stage that name the same path. The plumbing for several resident models is there — what is missing is the ability to *consult* more than one for the same utterance.

## Why that is limiting

A classifier's label head is frozen at train time (the module docstring says as much). That gives two useful but mutually exclusive shapes:

- a **broad multilingual model**, which covers any skill because the prototype store is built at runtime from whatever intents registered; and
- a **narrow fine-tuned classifier**, which is far more accurate on the intents it was trained on and useless on anything else.

Today a deployment picks one per language. Choosing the classifier means any skill installed after training is invisible to that stage; choosing the broad model means giving up the accuracy the fine-tune bought. Neither is the behaviour most deployments actually want, which is "use the specialised model where it applies, fall back to the general one everywhere else".

This is sharpest on fleets where the installed skill set differs per device: a label the classifier is confident about may name an intent that is not registered on *this* device at all.

## What would help

Let a language name an ordered list, not just a string:

```python
{"models": {"en": ["org/fine-tuned-intents", "org/general-multilingual"]}}
```

and consult them in order, with a couple of properties that matter:

- **A label the current session cannot route to is not an answer.** If the top model returns a label no registered intent matches, that is a miss for that model, not for the stage — move to the next one. `labels.json` already describes a model's label space, so this is checkable without asking the model twice.
- **Confidence has to stay comparable.** `CONF_THRESHOLDS` already carries separate scales for `classifier` and `prototype` precisely because a softmax and a cosine similarity are not the same number. Mixing two models in one stage needs the same care, per model rather than per mode.
- **One embedding, still.** Where two entries resolve to the same path, `_SHARED_MODELS` should keep doing what it does now and load the matrix once.

Backwards compatible: a plain string keeps today's meaning exactly, and a one-element list is the same thing.

## Alternatives considered

- *Two pipeline stages at different confidence levels* — possible now, but both stages pay a full prototype build and the ordering is expressed in the pipeline list rather than alongside the models, which makes the per-language part awkward to express.
- *Retraining the classifier whenever a skill is installed* — not workable where the installed set is per-device and changes at runtime.

Happy to put up a PR if the shape sounds right. The part I would want a maintainer's opinion on first is whether "unroutable label ⇒ try the next model" belongs in this plugin or in the caller.

**Comment by goldyfruit** [comment]
@openvoiceos-bot what do you think?
